src/main/kotlin/com/github/vol0n/utbotcppclion/actions/GenerateForAssertionAction.kt

```diff
@@ -0,0 +1,17 @@
+package com.github.vol0n.utbotcppclion.actions
+
+import com.github.vol0n.utbotcppclion.actions.utils.client
+import com.intellij.openapi.actionSystem.AnActionEvent
+import com.intellij.openapi.actionSystem.CommonDataKeys
+
+class GenerateForAssertionAction: UTBotTestsResponseAction() {
+    override val funToGetTestResponse = { e: AnActionEvent ->
+        client.generateForAssertion(buildAssertionRequestFromEvent(e))
+    }
+
+    override fun update(e: AnActionEvent) {
+        val project = e.project
+        val editor = e.getData(CommonDataKeys.EDITOR)
+        val file = e.getData(CommonDataKeys.VIRTUAL_FILE)
+        e.presentation.isEnabledAndVisible = (project != null) && (editor != null) && (file != null)    }
```

Comment:
Please use correct formatting. You have closing curly bracket on this line. Please move it to the next. You can autoformat code with intellij 
![image](https://user-images.githubusercontent.com/17502570/139542771-d784f877-14af-48cc-9a66-4e2792bb76df.png)